Misc/NEWS.d/next/Documentation/2017-11-19-13-34-28.bpo-32051.KBMEYJ.rst

```diff
@@ -0,0 +1,2 @@
+Remove name conflict under "Using a remove manager" section, the original
```

Comment:
No need to add a NEWS entry for trivial documentation changes. Please delete this file.